.travis.yml

```diff
@@ -47,6 +49,17 @@ install:
 before_script:
   - mysql -e 'create database olympia;'
   - node --version
+  - |
+      if [[ $TOXENV == "ui-tests" ]]; then
+        wget -O /tmp/geckodriver.tar.gz https://github.com/mozilla/geckodriver/releases/download/v0.15.0/geckodriver-v0.15.0-linux64.tar.gz
+        mkdir $HOME/geckodriver && tar xvf /tmp/geckodriver.tar.gz -C $HOME/geckodriver
+        export PATH=$HOME/geckodriver:$PATH
+        firefox --version
+        geckodriver --version
+        sh -e /etc/init.d/xvfb start
+        sleep 10
+        make update_deps
+      fi
```

Comment:
Future improvement idea (doesn't have to be done here), let's move this bash script to the `scripts` directory for potential re-use.